src/VirtualClient/VirtualClient.Actions/FIO/FioMetricsParser.cs

```diff
@@ -285,6 +288,19 @@ private void AddMeasurement(IList<Metric> metrics, JToken job, string path, stri
             }
         }
 
+        private void ParseFioVersion(List<Metric> metrics)
+        {
+            JToken resultsJson = JObject.Parse(this.RawText);
+
+            JToken fioVersionToken = resultsJson["fio version"];
+            if (fioVersionToken != null)
+            {
+                var metricMetaData = new Dictionary<string, IConvertible>();
+                metricMetaData["fio version"] = fioVersionToken.Value<string>();
+                metrics.Add(new Metric("IsFioVersionCaptured", 1, null, MetricRelativity.Undefined, verbosity: 1, metadata: metricMetaData));
```

Comment:
This is not a metric worth capturing. It is not a measurement of FIO outcomes but more precisely metadata from it. Put this in the metadata if you want but not in the metrics.